src/refspec.c

```diff
@@ -252,7 +252,7 @@ int git_refspec_transform(git_buf *out, const git_refspec *spec, const char *nam
 	}
 
 	if (!spec->pattern)
-		return git_buf_puts(out, spec->dst);
+		return git_buf_puts(out, spec->dst ? spec->dst : "");
```

Comment:
Do you want to write a test for it? I think it should be rather easy to do as there is already precedence of testing this function in "tests/network/refspecs.c".

Reply:
Done!